src/tv2-common/videoSwitchers/atemToTricasterDveConverter.ts

```diff
@@ -0,0 +1,68 @@
+import { TSR } from 'blueprints-integration'
+import { TriCasterDveConverter } from './TriCasterDveConverter'
+
+const ATEM_WIDTH = 32
+const ATEM_HEIGHT = 18
+
+const ATEM_CROP_LEFT_RIGHT_MAX_VALUE = 32000
+const ATEM_CROP_TOP_BOTTOM_MAX_VALUE = 18000
+
+const TRICASTER_WIDTH = (2 / 9) * 16
```

Comment:
Just to make it completely transparent, this can be refactored to:
```suggestion
const TRICASTER_HEIGHT = 2
const TRICASTER_WIDTH = TRICASTER_HEIGHT * (16 / 9)
```